### DISCUSSION TITLE
Unexpected input during declaring channel from file pairs

### DISCUSSION THREAD

**Discussion by zillurbmb51** [nextflow-io/nextflow#3783] [Q&A] [answered]
Hi,

I was declaring channel from file pairs and getting error `unexpected input`
My code:
`Channel.fromFilePairs("${params.out}*/*unrelated_imp.{bed,bim,fam}",size:3).set{extracted_imp3}`

The res of the code in the log file is commented out using `/* ...comment block ...*/`
Attached is the log file. I never got this type of error. Any help?
[nextflow.log2.txt](https://github.com/nextflow-io/nextflow/files/11028380/nextflow.log2.txt)

**Comment by bentsherman** [comment]
Looks like a Groovy syntax error. Can you post the script that you ran?

**Comment by zillurbmb51** [comment]
Thank you @bentsherman 
Attached is the original script.
[mostest_with_nextflow_v2.nf.txt](https://github.com/nextflow-io/nextflow/files/11030672/mostest_with_nextflow_v2.nf.txt)

**Comment by bentsherman** [comment, marked as answer]
I copied your script into VS Code with Groovy highlighting, it looks like the `*/` on this line:
```groovy
Channel.fromFilePairs("${params.out}*/*chunk*csv* ...
```

Is interpreted as a comment closer. I think it is a limitation of the Groovy parser. I would remove the commented code or disable it in a different way. For example, in VS Code you can highlight many lines and do Ctrl+/ to comment them out with line comments.

**Comment by bentsherman** [comment]
Alternatively, you might be able to escape the `*/*` as `*\/*`

**Comment by zillurbmb51** [comment]
Thank you. It works now!
